<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainForm</class>
 <widget class="QWidget" name="TrainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="StartpushButton">
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>选择需要训练的图集,训练过程比较长,请耐心等候,训练结束后可通过启动服务按钮，在web端上传需要识别的验证码</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="ServerpushButton">
     <property name="text">
      <string>启动服务</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="ChooseFilepushButton">
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
